Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -663,7 +663,7 @@ class MessageDetailViewController: OWSViewController, MediaGalleryDataSourceDele
 
         if let audioAttachmentPlayer = self.audioAttachmentPlayer {
             // Is this player associated with this media adapter?
-            if audioAttachmentPlayer.owner as? ConversationViewItem == viewItem {
+            if audioAttachmentPlayer.owner === viewItem {
```

Comment:
This was also not a mechanical change - ultimately what we want is reference equality, so this is simpler.